.github/workflows/go.yml

```
@@ -4,16 +4,35 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.19', '1.20' ]
     steps:
     - name: Install Go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
       with:
-        go-version: ~1
+        go-version: ${{ matrix.go }}
         check-latest: true
       id: go
     - name: Tooling
       run: go install gotest.tools/gotestsum@latest
+
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+
     - name: Test
-      run: gotestsum
+      run: gotestsum -- -coverpkg=./internal/database,./internal/model,./internal/scheduler,./internal/storage,./internal/webserver,./internal/webserver/middleware,./internal/webserver/serializer,./internal/webserver/service,./internal/webserver/weberror,./internal/xpath,./tests -coverprofile=coverage.out ./...
```

It should be better to add a dynamic argument because if we add new packages, the coverage will not take it.

```suggestion
      run: gotestsum -- -coverpkg=./... -coverprofile=coverage.out ./...
```